release/create-release-tag.sh

```diff
@@ -133,37 +133,37 @@ update_code() {
   # Not all operators have a getting started guide
   # that's why we verify if templating_vars.yaml exists.
   if [ -f "$1/docs/templating_vars.yaml" ]; then
-    yq -i "(.versions.[] | select(. == \"*dev\")) |= \"${RELEASE}\"" "$1/docs/templating_vars.yaml"
+    yq -i "(.versions.[] | select(. == \"*dev\")) |= \"${RELEASE_TAG}\"" "$1/docs/templating_vars.yaml"
     yq -i ".helm.repo_name |= sub(\"stackable-dev\", \"stackable-stable\")" "$1/docs/templating_vars.yaml"
     yq -i ".helm.repo_url |= sub(\"helm-dev\", \"helm-stable\")" "$1/docs/templating_vars.yaml"
   fi
 
   #--------------------------------------------------------------------------
   # Replace .spec.image.stackableVersion for getting-started examples.
-  # N.B. yaml files should contain a single document.
+  # This should be done by using the templating script as the paths
+  # are not always consistent.
   #--------------------------------------------------------------------------
-  if [ -d "$1/docs/modules/getting_started/examples/code" ]; then
-    for file in $(find "$1/docs/modules/getting_started/examples/code" -name "*.yaml"); do
-      yq -i "(.spec | select(has(\"image\")).image | (select(has(\"stackableVersion\")).stackableVersion)) = \"${RELEASE}\"" "$file"
-    done
-  fi
+  #if [ -d "$1/docs/modules/getting_started/examples/code" ]; then
+  #  for file in $(find "$1/docs/modules/getting_started/examples/code" -name "*.yaml"); do
+  #    yq -i "(.spec | select(has(\"image\")).image | (select(has(\"stackableVersion\")).stackableVersion)) = \"${RELEASE}\"" "$file"
+  #  done
+  #fi
```

Comment:
Do we need this again? If not i would say remove.

Reply:
Done (removed)